.travis.yml

```diff
@@ -1,20 +1,15 @@
+dist: trusty
```

Comment:
I don't think you need this. It's yet another thing we will forget to change later on. Leaving it out means you will run on latest (Xenial) which is just fine for the versions you need.